client-spark/common/src/main/java/org/apache/uniffle/shuffle/manager/RssShuffleManagerBase.java

```diff
@@ -638,7 +638,7 @@ public ShuffleHandleInfo getShuffleHandleInfoByShuffleId(int shuffleId) {
   @Override
   public int getMaxFetchFailures() {
     final String TASK_MAX_FAILURE = "spark.task.maxFailures";
-    return Math.max(1, sparkConf.getInt(TASK_MAX_FAILURE, 4) - 1);
+    return Math.max(0, sparkConf.getInt(TASK_MAX_FAILURE, 4) - 1);
```

Comment:
I think the original 1 is wrong if the `spark.task.max.failure=1`

Reply:
Could you add this into the comment?

Reply:
After rethinking this part code, I think this need to be optimized in the next PRs. the detailed problems is shown in #1798 #1801 . So the comment may be not necessary.

Reply:
OK.